app/components/drafts/drafts.js

```diff
@@ -13,23 +14,27 @@ const StyledAddContent = styled(AddContent)`
   top: -2px;
 `
 
-export default ({ p2p }) => {
-  const [modules, setModules] = useState()
+export default ({ p2p, profile }) => {
+  const [drafts, setDrafts] = useState()
+  const [hasListedContent, setHasListedContent] = useState()
 
   useEffect(() => {
     ;(async () => {
-      setModules(await p2p.listContent())
+      const modules = await p2p.listContent()
+      const drafts = modules.filter(mod => !isModuleListed(mod, profile))
+      setHasListedContent(profile.rawJSON.contents.length > 0)
```

Comment:
```suggestion
      setHasListedContent(profile.rawJSON.contents.length > 0)
      const modules = await p2p.listContent()
      const drafts = modules.filter(mod => !isModuleListed(mod, profile))
```

reordered because the `setHasListedContent` call has nothing to do with the two lines above